content/blog/2021-10-05-extending-models-with-more-data.md

```diff
@@ -0,0 +1,263 @@
+
+We'll add the `train` data for `dogs` to the corresponding directory in
+`data/hymenoptera_data` just like we did for the `cats` data. Now we can run a
+new experiment with all of the new data included. We'll still need to reset the
+experiment like before, so run the following command.
+
+```dvc
+dvc exp run --reset
+```
+
+Once the training epochs are finished, we can take one more look at that metrics
+table.
+
+```dvc
+dvc exp show --no-timestamp --include-metrics step,acc,val_acc,loss,val_loss --include-params lr,momentum
+```
+
+Now we'll have all three experiments to compare.
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━┳━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ metric:**step** ┃     metric:**acc** ┃ metric:**val_acc** ┃    metric:**loss** ┃ metric:**val_loss** ┃ param:**lr**    ┃ param:**momentum** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━╇━━━━━━━━━━┩
+│ **workspace**               │    **3** │  **0.8795** │ **0.90667** │ **0.29302** │  **0.25752** │ **0.001** │ **0.09**     │
+│ **data-change**             │    **-** │       **-** │       **-** │       **-** │        **-** │ **0.001** │ **0.09**     │
+│ │ ╓ c20220f [exp-82f70] │    3 │  0.8795 │ 0.90667 │ 0.29302 │  0.25752 │ 0.001 │ 0.09     │
+│ │ ╟ fcb5a0b             │    2 │ 0.85915 │ 0.92333 │ 0.38274 │  0.25257 │ 0.001 │ 0.09     │
+│ │ ╟ 3768821             │    1 │ 0.80751 │ 0.84667 │ 0.47681 │  0.40228 │ 0.001 │ 0.09     │
+│ ├─╨ 7e1b8fb             │    0 │ 0.64632 │    0.84 │ 0.87301 │  0.46744 │ 0.001 │ 0.09     │
+│ │ ╓ 9405575 [exp-54e8a] │    3 │ 0.91389 │    0.87 │ 0.20506 │  0.66306 │ 0.001 │ 0.09     │
+│ │ ╟ 856d80f             │    2 │ 0.90215 │ 0.87333 │ 0.27204 │  0.61631 │ 0.001 │ 0.09     │
+│ │ ╟ 23dc98f             │    1 │ 0.87671 │    0.86 │ 0.35964 │  0.61713 │ 0.001 │ 0.09     │
+│ ├─╨ 99a3c34             │    0 │ 0.71429 │    0.82 │ 0.67674 │  0.62798 │ 0.001 │ 0.09     │
+│ │ ╓ 3b3a2a2 [exp-23593] │    3 │ 0.86885 │    0.46 │ 0.31573 │   3.7067 │ 0.001 │ 0.09     │
+│ │ ╟ 93d015d             │    2 │ 0.83197 │ 0.41333 │ 0.36851 │   3.4259 │ 0.001 │ 0.09     │
+│ │ ╟ d474c42             │    1 │ 0.79918 │ 0.43333 │ 0.46612 │    3.286 │ 0.001 │ 0.09     │
+│ ├─╨ 1582b4b             │    0 │ 0.52869 │    0.39 │ 0.94102 │   2.5967 │ 0.001 │ 0.09     │
+└─────────────────────────┴──────┴─────────┴─────────┴─────────┴──────────┴───────┴──────────┘
+```
+
+These results make sense for the experiments we've run. We're paying attention
+to the validation accuracy here because this gives us a fair comparison of
+what's happening as we add more data.
+
+The first experiment's training metrics are for bees and ants. The second
+experiment's training metrics are for bees, ants, and cats. And the third
+experiment's training metrics are for all four classes. So we can't really
+compare these metrics. That's why it looks like this last experiment has some
```

Comment:
I would drop this last sentence. I think of overfitting as being more like the first experiment, where training metrics are high but validation metrics are low because the model is overfit to the specific data included in training and doesn't fit other data well.

Reply:
Gotcha. Thanks for all of these explanations on this post! They've been super helpful for my general understanding of things. 😄